proot-distro.sh

```diff
@@ -814,6 +814,54 @@ command_remove_help() {
 	show_version
 	msg
 }
+command_clear_cache_help() {
+	msg
+	msg "${BYELLOW}Usage: ${BCYAN}$PROGRAM_NAME ${GREEN}clear-cache"
+	msg
+	msg "${CYAN}This command will delete the download cache...${RST}"
+	msg
+	msg "${CYAN}This command is useful to clear some space by deleting dowmloaded rootfs tarballs${RST}"
+	msg
+	show_version
+	msg
+}
+
+#############################################################################
+#
+# FUNCTION TO CLEAR DLCACHE
+#
+
+command_clear_cache() {
+	if [ $# -ge 1 ]; then
+		case "$1" in
+			-h|--help) command_clear_cache_help;;
+			*)
+				msg
+				msg "${BRED}Error: unknown command '${YELLOW}$1${BRED}'.${RST}"
+				msg
+				msg "${CYAN}Run '${GREEN}${PROGRAM_NAME} clear-cache --help${CYAN}' to see the list of available commands.${RST}"
+				msg
+				exit 1
+			;;
+		esac
+	elif (( $# == 0 )); then
+		if ! ls -la "${DOWNLOAD_CACHE_DIR}"/*tar.gz > /dev/null 2>&1; then
+			msg "${BLUE}[${GREEN}*${BLUE}] ${CYAN}No Download cache found...${RST}"
+		else
+			SIZE_OF_CACHE="$(du -d 0 -h -a ${DOWNLOAD_CACHE_DIR} | awk '{$2=$2};1' | cut -d " " -f 1)"
+			msg "${BLUE}[${GREEN}*${BLUE}] ${CYAN}Clearing cache files...${RST}"
+			ls "${DOWNLOAD_CACHE_DIR}"/*tar.gz > temp_buffer.txt
+
+			for f in $(cat temp_buffer.txt); do
+				msg "${BLUE}[${GREEN}*${BLUE}] ${CYAN} Removing ${BLUE} ${f} ${RST}"
+				rm -rf "${f}"
+			done
```

Comment:
Use `find` + `while` loop instead of `ls` and unnecessary "buffer" file.

Reply:
I think `for` loop is good for this case